### ISSUE TITLE
[A11y] Explore Collections - Grid - Pagination (multiple issues)

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1894]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Pagination fails WCAG in a few ways:

- There's no meaningful context
- The "Prev" and "Next" links, when disabled, are still keyboard-focusable.  This makes it confusing because you can't actually activate the link.
- The current page isn't focusable.  This may not be a hard failure, but it seems pretty common to have a focusable current page that uses `aria-current="true"`

### Suggested Fix

https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Pagination is a good resource.  I think it mentions all of the following:

- A navigation region (`<nav>`) with a label of "Pagination" would suffice for context
- Either make disabled links non-focusable or make sure their state is clearly communicated to assistive technology (a.g., `aria-disabled="true"`)
- Consider making the current page activatable just so it's more consistent, and use ARIA to mark it up.  When a nav item isn't a link, screen reader users will never see it.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

This may need a screen reader user to test, otherwise it's a fairly error-prone situation of looking closely at the markup to make sure it seems to follow what MDN recommended.

**Comment by CGillen** [comment]
@thornka Focus on the last two failures. They both relate to `tabindex` and `aria` (as described by Jeremy). If you want to tackle the first failure, Jeremy has laid out a suggested fix but feel free to get me for a pair session.
